enigma/src/main/java/cuchaz/enigma/translation/mapping/serde/tinyv2/TinyV2Reader.java

```diff
@@ -111,19 +110,17 @@ private EntryTree<EntryMapping> read(Path path, List<String> lines, ProgressList
 
 						if (state.get(IN_CLASS)) {
 							switch (parts[0]) {
-								case "m": // method
+								case "m" -> { // method
 									state.set(IN_METHOD);
 									holds[IN_METHOD] = parseMethod(holds[IN_CLASS], parts, escapeNames);
-									break;
```

Comment:
Why have all the breaks been removed from this switch? Have you verified they're unnecessary?

Reply:
I believe they are not required with the new switch-case syntax

Reply:
Interesting, I'm not convinced

Reply:
Yeah, switch expressions don't need breaks